server/db/assessmentsDb.js

```diff
@@ -0,0 +1,20 @@
+const environment = 'development'
```

Comment:
Please utilise the database config already built into `db/index.js`. This is to avoid duplicated code as we will need the same config in each new db module.